reference/build_helpers/cmake.rst

```diff
@@ -133,6 +133,23 @@ An example of the value of this property could be:
 
     --config Release
 
+parallel
+++++++++
+
+**Defaulted to**: ``True``
+
+Run CMake process in parallel for compilation, installation and testing. This attribute gets the value for the class constructor and
+appends the ``/MP`` flag in the case of the Visual compiler. However, the parallel executing can be changed for testing for example:
+
+.. code-block:: python
+
+    cmake = CMake(self)
```

Comment:
I would add the call to `cmake.configure()`, just one line and it turns the snippet into a copy-paste valid one.